full-service/src/service/account.rs

```diff
@@ -17,7 +17,57 @@ use mc_fog_report_validation::FogPubkeyResolver;
 use mc_ledger_db::Ledger;
 use mc_util_from_random::FromRandom;
 
+use crate::service::ledger::LedgerServiceError;
 use diesel::Connection;
+use displaydoc::Display;
+
+#[derive(Display, Debug)]
+pub enum AccountServiceError {
```

Comment:
Opportunistically isolated AccountServiceErrors - no change in functionality here.